src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  min-width: 16px;
+  min-height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars__star-active {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
we can target certain children of `stars` class via `:nth-child(number)` pseudoclass, and then target children of every `star`
element via `:nth-child(n + number)` pseudoclass - although your solution works great, I would like you to try this one out too 😄